AdminSelfUpgrade.php

```diff
@@ -2905,51 +2898,59 @@ public function upgradeThisFile($file)
-                    $this->nextQuickInfo[] = sprintf($this->l('[WARNING] File %1$s has been deleted.'), $file);
+                    $this->nextQuickInfo[] = $this->trans('[WARNING] File %1$s has been deleted.', array($file), 'Modules.Autoupgrade.Admin');
                 }
                 if (!file_exists($dest)) {
                     if (mkdir($dest)) {
-                        $this->nextQuickInfo[] = sprintf($this->l('Directory %1$s created.'), $file);
+                        $this->nextQuickInfo[] = $this->trans('Directory %1$s created.', array($file), 'Modules.Autoupgrade.Admin');
                         return true;
                     } else {
                         $this->next = 'error';
-                        $this->nextQuickInfo[] = sprintf($this->l('Error while creating directory %s.'), $dest);
-                        $this->nextErrors[] = $this->next_desc = sprintf($this->l('Error while creating directory %s.'), $dest);
+                        $this->nextQuickInfo[] = $this->trans('Error while creating directory %s.', array($dest), 'Modules.Autoupgrade.Admin');
+                        $this->nextErrors[] = $this->next_desc = $this->trans('Error while creating directory %s.', array($dest), 'Modules.Autoupgrade.Admin');
                         return false;
                     }
                 } else { // directory already exists
-                    $this->nextQuickInfo[] = sprintf($this->l('Directory %1$s already exists.'), $file);
+                    $this->nextQuickInfo[] = $this->trans('Directory %s already exists.', array($file), 'Modules.Autoupgrade.Admin');
                     return true;
                 }
             } elseif (is_file($orig)) {
                 if ($this->isTranslationFile($file) && file_exists($dest)) {
                     $type_trad = $this->getTranslationFileType($file);
                     $res = $this->mergeTranslationFile($orig, $dest, $type_trad);
                     if ($res) {
-                        $this->nextQuickInfo[] = sprintf($this->l('[TRANSLATION] The translation files have been merged into file %1$s.'), $dest);
+                        $this->nextQuickInfo[] = $this->trans('[TRANSLATION] The translation files have been merged into file %s.', array($dest), 'Modules.Autoupgrade.Admin');
                         return true;
                     } else {
-                        $this->nextQuickInfo[] = sprintf($this->l('[TRANSLATION] The translation files have not been merged into file %1$s. Switch to copy %2$s.'), $dest, $dest);
-                        $this->nextErrors[] = sprintf($this->l('[TRANSLATION] The translation files have not been merged into file %1$s. Switch to copy %2$s.'), $dest, $dest);
+                        $this->nextQuickInfo[] = $this->trans(
+                            '[TRANSLATION] The translation files have not been merged into file %filename%. Switch to copy %filename%.',
+                            array('%filename%' => $dest),
```

Comment:
Yes. It means that it will try to merge the new files and the older one into %filename%. If it fails, it will simply copy it.